pkgs/by-name/co/corto/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch,
+  cmake,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "corto";
+  version = "unstable-2023-06-27";
+
+  src = fetchFromGitHub {
+    owner = "cnr-isti-vclab";
+    repo = finalAttrs.pname;
```

```suggestion
    repo = "corto";
```
If pname is appended with something like `-headless` or `-patched` this doesn't make sense anymore.